conanfile.txt

```diff
@@ -1,33 +1,36 @@
 [requires]
-cli11/1.9.1
-date/27d1e1e@ess-dmsc/stable
-eigen/3.3.9
-fmt/6.2.0
+cli11/2.1.1      #bump to 2.1.2 requires conan upgrade
+date/3.0.1
+eigen/3.3.9      #bump to 3.4.0 requires conan upgrade
 gtest/1.10.0
-h5cpp/0.3.3@ess-dmsc/stable
-nlohmann_json/3.9.1
-librdkafka/1.6.0
+fmt/6.2.0        #bump to 8.1.1 requires upgrade of spdlog as well
+librdkafka/1.6.0 #bump to 1.8.2 requires conan upgrade
+nlohmann_json/3.10.5
+spdlog/1.8.5     #bump to 1.9.2 requires conan upgrade
+h5cpp/0.4.1@ess-dmsc/stable
 logical-geometry/705ea61@ess-dmsc/stable
-qplot/2ffc74f@ess-dmsc/stable
-Qt-Color-Widgets/a95f72e@ess-dmsc/stable
-spdlog/1.8.5
+qplot/6e192ab@ess-dmsc/stable
+Qt-Color-Widgets/9f4e052@ess-dmsc/stable
 streaming-data-types/6a41aee@ess-dmsc/stable
 
 [generators]
 cmake
-virtualrunenv
 cmake_find_package
+virtualbuildenv
+virtualrunenv
 
 [options]
 gtest:shared=False
 librdkafka:shared=False
+flatbuffers:shared=True
+hdf5:shared=True
+date:use_system_tz_db=True
+h5cpp:with_boost=False
 
 [imports]
-bin, cmake -> ./bin
-bin, *.dll -> ./bin
-lib, *.dll -> ./bin
-lib, *.so.* -> ./lib
-lib, *.so -> ./lib
-share, * -> ./share
-filesystem/lib, *.so* -> ./lib
-system/lib, *.so* -> ./lib
+., *.dylib* -> ./lib @ keep_path=False
+., *.so* -> ./lib @ keep_path=False
+., LICENSE* -> ./licenses @ folder=True, ignore_case=True
+., *.dll -> ./bin @ keep_path=False
+., *.a* -> ./lib @ keep_path=False
+., *.lib -> -> ./bin @ keep_path=False
```

Comment:
I really have no idea what is going on here so I just assume this is correct, but
how about the double arrow at the end?

Reply:
Actually, I am not sure what is going on here either :D
I think I copied this section from either kafka-to-nexus just to keep in step with what your latest practice was. I have never understood this section too well. Perhaps @amues has some insight on what the correct thing to do here is?

Reply:
Fair enough. I'll check this out with the kafka-to-nexus people. I think the double arrow is an error. If there is nothing else coming out of this I will go ahead and squash, merge and delete the branch sometime today or tomorrow.